packages/zendesk-adapter/src/filters/article/article.ts

```diff
@@ -201,8 +201,8 @@ const handleArticleAttachmentsPreDeploy = async ({ changes, client, elementsSour
   const attachmentChanges = changes
     .filter(isAdditionOrModificationChange)
     .filter(change => getChangeData(change).elemID.typeName === ARTICLE_ATTACHMENT_TYPE_NAME)
-  await awu(attachmentChanges)
-    .forEach(async attachmentChange => {
+  const attachmentsPromises = attachmentChanges
```

Comment:
1. why not just 
```suggestion
  await Promise.all(...)
```
here?

2. how did we test / benchmark this, and how much of the flow was covered? I'm not sure we should parallelize everything since the `associateAttachments` call might have some race conditions we're not aware of. we can probably parallelize the `createUnassociatedAttachment` part but that requires some refactoring